### ISSUE TITLE
Malformed NOTIFY responses

### ISSUE THREAD

**Issue by hotab** [MoshiBin/ssdpy#47]
Per https://openconnectivity.org/upnp-specs/UPnP-arch-DeviceArchitecture-v2.0-20200417.pdf

```
Note: No body is sent for messages with method NOTIFY, but note that the message shall
have a blank line following the last header field.
```

The library does not seem to add a blank line.

**Comment by hotab** [comment]
PR: https://github.com/MoshiBin/ssdpy/pull/48

**Comment by MoshiBin** [comment]
Fixed in #48, closing this
